addons/website/static/src/js/content/snippets.animation.js

```diff
@@ -858,6 +858,25 @@ registry.socialShare = Animation.extend({
     },
 });
 
+registry.facebookPage = Animation.extend({
+    selector: '.o_facebook_page',
+
+    /**
+     * @override
+     */
+    start: function () {
+        var params = _.pick(this.$el.data(), 'href', 'adapt_container_width', 'height', 'width', 'tabs', 'small_header', 'hide_cover', 'show_facepile');
+        if (params && params.href) {
+            var actualWidth = this.$el.width();
+            params.width = actualWidth > 500 ? 500 : actualWidth < 180 ? 180 : actualWidth;
+            var newSrc = $.param.querystring('https://www.facebook.com/plugins/page.php', params);
+            this.$('iframe').remove();
```

Comment:
The <iframe/> should not be saved so that we do not have to remove it on page loading. The <iframe/> needs to be remove in the `destroy` method of the animation.